src/styles/borders.js

```diff
@@ -1,7 +1,7 @@
 import { upperFirst } from 'lodash';
-import { css } from 'styled-components';
 import colors from './colors';
 import position from './position';
+import { css } from 'rainbowed-components';
```

Comment:
Should `css` still be imported from `styled-components` since it's being passed a string?

Reply:
It's re-exported from styled components. for backward compatibility with some existing code left what uses that css thing. It's something we want to remove in the future completely.